src/components/radio/Radio.tsx

```diff
@@ -0,0 +1,46 @@
+import { Component, createSignal, Show } from 'solid-js'
+
+import StyledIcon from './StyledIcon'
+import StyledInput from './StyledInput'
+import StyledLabel from './StyledRadio'
+
+export type IRadioProps = {
+  label?: any
```

Comment:
Maybe `label?: JSX.Element` instead of `any`? (importing JSX from solid-js)

Reply:
Yep, I was not sure so I added any. It can be Element, Component or text etc so I have use any for now. Will update it